Formula/rtags.rb

```diff
@@ -4,6 +4,7 @@ class Rtags < Formula
   url "https://github.com/Andersbakken/rtags.git",
       :tag => "v2.2",
```

Comment:
There's a 2.3 tag of this now, so might want to do that instead of the revision.